config/conn.php

```diff
@@ -1,8 +1,9 @@
 <?php
 $servername = "localhost";
```

Comment:
Please use PSR-1 camel case, ask @peerfahad  what is it. Or simply read it http://www.php-fig.org/psr/psr-1/#42-properties